src/state/slices/opportunitiesSlice/constants.ts

```diff
@@ -137,3 +137,9 @@ export const DEFI_PROVIDER_TO_METADATA: Record<DefiProvider, DefiProviderMetadat
     url: 'https://app.shapeshift.com',
   },
 }
+
+export const isRunePoolOpportunity = (
+  opportunity: any,
```

Comment:
ser

Reply:
Goddamn 😆

Reply:
`UserStakingOpportunity` pls 🙏🏽

Reply:
The real type behind was `StakingEarnOpportunityType`, I finally used it along with `RunepoolUserStakingOpportunity` in [231aabd](https://github.com/shapeshift/web/pull/7472/commits/231aabd2822d7bf58613c7fd32ba5873c67846a8)